### ISSUE TITLE
CALL TRANSFORMATION

### ISSUE THREAD

**Issue by larshp** [abaplint/transpiler#253]
```abap
    CALL TRANSFORMATION id
      OPTIONS value_handling = 'accept_data_loss'
      SOURCE XML lv_xml
      RESULT data = rs_data.
```

```abap
    CALL TRANSFORMATION id
      OPTIONS initial_components = 'suppress'
      SOURCE data = is_data
      RESULT XML rv_xml.
```

https://github.com/abapGit/abapGit/blob/master/src/zcl_abapgit_dot_abapgit.clas.abap

**Comment by larshp** [comment]
time to start this one 😅

**Comment by larshp** [comment]
note: the actual logic will be implemented via open-abap/open-abap#142, this PR just hooks it together
